sycl/test-e2e/Matrix/joint_matrix_opt_kernel_feature_unsupported_hw.cpp

```diff
@@ -6,9 +6,7 @@
 // Test checks that exception will be thrown in case object of joint_matrix type
 // is used on unsupported HW, in this case, on Gen12.
 
-#include <sycl/sycl.hpp>
-
-using namespace sycl::ext::oneapi::experimental::matrix;
+#include "common.hpp"
```

Comment:
This test is running only on gen12, please, check that it is passing after your change.

Reply:
Passed see below.